src/ManipulationMsg.chpl

```diff
@@ -1259,4 +1259,174 @@ module ManipulationMsg {
     const ret = [i in d] gEnts[i]!;
     return ret;
   }
+
+  // https://numpy.org/doc/stable/reference/generated/numpy.delete.html
+  @arkouda.registerND
+  proc deleteMsg(cmd: string, msgArgs: borrowed MessageArgs, st: borrowed SymTab, param nd: int): MsgTuple throws {
+    param pn = Reflection.getRoutineName();
+    const arr = msgArgs.getValueOf("arr"),
+          obj = msgArgs.getValueOf("obj"),
+          objSorted = msgArgs.get("obj_sorted").getBoolValue(),
+          axis = msgArgs.get("axis").getPositiveIntValue(nd),
+          rname = st.nextName();
+
+    var gArr: borrowed GenSymEntry = getGenericTypedArrayEntry(arr, st),
+        gObj: borrowed GenSymEntry = getGenericTypedArrayEntry(obj, st);
+
+    proc doDelete(type t): MsgTuple throws
+      where nd == 1
+    {
+      const eIn = toSymEntry(gArr, t, 1),
+            eObj = toSymEntry(gObj, int, 1);
+
+      var eOut = st.addEntry(rname, eIn.a.size - eObj.a.size, t);
+
+      // copy the data from the input array to the output array, excluding the indices in 'obj'
+      // TODO: fold this loop into 'parSearch' (avoid repeatedly spawning tasks by kicking
+      //       off coforall loops for each array element)
```

Comment:
Just double checking that this is meant to be future work and not something that was forgotten